packages/components/pie-link/test/visual/pie-link.spec.ts

```diff
@@ -51,7 +50,7 @@ test.beforeEach(async ({ mount }, testInfo) => {
 
 componentVariants.forEach((variant) => test(`should render all prop variations for Variant: ${variant}`, async ({ page, mount }) => {
     for (const combo of componentPropsMatrixByVariant[variant]) {
-        const testComponent: WebComponentTestInput = createTestWebComponent(combo, renderTestPieLink);
```

Comment:
Why this one was changed?

Reply:
Hey, the type is already inferred from the return type of the function, so I think adding it here was causing some minor clutter.

Reply:
@fernandofranca Hey, the type is already being inferred from the return type of the function, so I think duplicating it here was causing some minor clutter.